src/App.tsx

```diff
@@ -103,17 +109,25 @@ export default function App() {
             />
           </Grid>
         </Grid>
-        <Grid container justify="flex-end">
+
+        <Grid container justify="space-between">
+          <Button
+            variant="contained"
+            color="primary"
```

Comment:
```suggestion
```

Let's make this button a different color (the default gray) by removing the `color` attribute